### ISSUE TITLE
formula that depends on cask

### ISSUE THREAD

**Issue by henridwyer** [geerlingguy/mac-dev-playbook#39]
How could I use this role to install a formula that depends on certain casks?

In my case I want to install wine, which requires the cask xquartz. I imagine there could be other similar cases.

**Comment by geerlingguy** [comment]
@henridwyer - You should be able to add taps to `homebrew_taps`, and casks to `homebrew_cask_apps` (see: https://github.com/geerlingguy/ansible-role-homebrew#role-variables)

**Comment by henridwyer** [comment]
Thanks for the answer. I will try to clarify the issue: if I try to install wine
```
homebrew_installed_packages:
  - wine

homebrew_cask_apps:
  -  xquartz
```

this will fail because the role tries to install wine **before** xquartz but can't because xquartz is a requirement.

What would be required is to specify the installation order (e.g. install this cask before this package). Is this possible?

Thanks for the help!

**Comment by oxyc** [comment]
I doubt there's ever a case when a cask depends on a package. They should all be self-contained, right? Maybe just move the cask task before the package tasks?

**Comment by geerlingguy** [comment]
I vaguely remember putting things in the order they are now because there _was_ some dependency issue in the reverse order...

**Comment by geerlingguy** [comment]
See: https://github.com/geerlingguy/ansible-role-homebrew/pull/7

**Comment by henridwyer** [comment]
it seems like that issue relates to the order of taps vs installing casks, not the order of installing formulas vs casks ?

**Comment by elliotweiser** [comment]
Fixed in geerlingguy/ansible-role-homebrew#58 and released in version [2.3.2](https://github.com/geerlingguy/ansible-role-homebrew/releases/tag/2.3.2) of the homebrew role.
